modules/pel/peltool/peltool.py

```diff
@@ -773,6 +794,16 @@ def main():
         parsePelFromBmcID(PELsPath, args.bmcID, config)
         sys.exit(0)
 
+    if args.plID:
+        config.plid = args.plID
+        parsePelFromPLIdAndSRCId(PELsPath, config)
+        sys.exit(0)
+
+    if args.srcID:
+        config.srcid = args.srcID
```

Comment:
Since the SRCID is 32 ASCII characters, there's no need to validate it like we do with PEL ID or PLID.

However, I believe that prefix matching on SRC would be more beneficial for filtering at the firmware subsystem level (e.g., BMC, HB, PHYP) or at the subsystem component level (e.g., [B_component_ids](https://github.com/openbmc/phosphor-logging/blob/master/extensions/openpower-pels/registry/B_component_ids.json) for Hostboot components or [O_component_ids](https://github.com/openbmc/phosphor-logging/blob/master/extensions/openpower-pels/registry/O_component_ids.json) for BMC components).

This would also allow for exact SRC match filtering to retrieve PELs.

Personally, I've found this useful in the past for obtaining PHAL-specific PELs.

@spinler Any thoughts?